client.go

```diff
@@ -966,15 +966,44 @@ func (f *File) readChunkAt(ch chan result, b []byte, off int64) (n int, err erro
 	return
 }
 
+func (f *File) readAtSequential(b []byte, off int64) (int, error) {
+	readed := 0
```

Comment:
I know sometimes we break English language rules to make terms more regular, but “written” and “read” should be the names of the values returned.

I would also recommend in this case naming the return arguments. (Even though, below, we’ll just return the explicit values.) Naming it as the return value makes it clear that we’re intending to reduce/accumulate the answer into this value.

Reply:
In my case I break English language rules, because I don't know english very well :)